README.md

```diff
@@ -66,3 +67,16 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing the App
+
+If you are experiencing issues with the App after making your own changes, you'll find two unit tests are ready to run to test that the View_model class is sorting the items as intended. You'll also find an integration test intended to test the app.py code base itself and the Trello API calls it makes. 
+
+If not already install, you'll need pytest, you can do this by running the command(please not if your using a virtual machine you may need to install this every time you start a new session):
+```bash
+$ pip install pytest 
+```
```

Comment:
This _shouldn't_ be true; you've added pytest to your dependencies tracked with poetry (see the `pyproject.toml` file) so it's picked up automatically when you run `poetry install`. Since it's in your virtual environment, pip won't recognise it as globally installed (so `pip install pytest` _will_ report it's freshly installing pytest each time) but it is already picked up and can be run by `poetry run pytest` even if pip doesn't recognise it.